internal/util/context.go

```diff
@@ -0,0 +1,26 @@
+package util
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+type contextKey string
+
+// Context keys.
+const (
+	contextKeyLogger = contextKey("logger")
+)
```

Comment:
I actually had a better idea. You can do this:

`type contextKeyLogger struct {}`

and use `var key contextKeyLogger;` as the key.